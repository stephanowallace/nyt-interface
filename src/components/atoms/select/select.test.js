import React from 'react';
import { shallow } from 'enzyme';
import selectOptions from './__fixture__/selectOptions';
import Select from '.';

describe('Select', () => {
  it('renders default state', () => {
    const props = {
      options: selectOptions,
      onChange: jest.fn(),
      selectedValue: selectOptions[0],
    };
    const component = shallow(<Select {...props} />);
    expect(component).toMatchSnapshot();
  });
});
