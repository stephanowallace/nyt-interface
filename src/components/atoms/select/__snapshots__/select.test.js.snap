// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select renders default state 1`] = `
<div
  className="container "
>
  <StateManager
    components={
      Object {
        "DropdownIndicator": null,
        "IndicatorSeparator": null,
      }
    }
    defaultInputValue=""
    defaultMenuIsOpen={false}
    defaultValue={null}
    menuPlacement="top"
    onChange={[MockFunction]}
    options={
      Array [
        Object {
          "label": "1",
          "value": "1",
        },
        Object {
          "label": "2",
          "value": "2",
        },
        Object {
          "label": "3",
          "value": "3",
        },
        Object {
          "label": "4",
          "value": "4",
        },
      ]
    }
    placeholder=""
    styles={
      Object {
        "control": [Function],
      }
    }
    value={
      Object {
        "label": "1",
        "value": "1",
      }
    }
  />
</div>
`;
